pontoon/base/models.py

```diff
@@ -109,16 +109,20 @@ def user_profile_url(self):
 
 def user_gravatar_url(self, size):
     email = hashlib.md5(self.email.lower().encode("utf-8")).hexdigest()
-    data = {"s": str(size)}
+    data = {"s": str(size), "d": str(404)}
```

Comment:
Instead of `404`, you can set the ui-avatars.com URL directly (see https://eu.ui-avatars.com/):

```python
data = {
    "s": str(size),
    "d": "https://ui-avatars.com/api/{name}/{size}/{background}/{color}".format(
        name=quote(self.display_name),
        size=size,
        background="333941",
        color="FFFFFF",
    ),
}
```

It would also help to use consistent styling.